src/libutil/url.hh

```diff
@@ -49,6 +49,10 @@ const static std::string pathRegex = "(?:" + segmentRegex + "(?:/" + segmentRege
 const static std::string refRegexS = "[a-zA-Z0-9][a-zA-Z0-9_.-]*"; // FIXME: check
 extern std::regex refRegex;
 
+// A github or gitlab url
+const static std::string urlRegexS = "[a-zA-Z0-9.]*"; // FIXME: check
```

Comment:
The name `urlRegex` does not correspond with that this actually is. Maybe it's better to move this to `github.cc`.

Reply:
moved it to `github.cc`